apps/src/dropletUtils.js

```diff
@@ -157,41 +159,47 @@ standardConfig.categories = {
 };
 
 /**
- * @param codeFunctions
+ * @param codeFunctions {object} A collection of named key/value pairs
+ *   key is a block name from dropletBlocks or standardBlocks
+ *   value is an object that can be used to override block defaults
  * @param {DropletConfig} dropletConfig
  * @param {DropletConfig} otherConfig optionally used to supply a standardConfig
  *  object which is not app specific. It will be used first, then overriden
  *  by the primary dropletConfig if there is overlap between the two.
  * @param paletteOnly boolean: ignore blocks not in codeFunctions palette
- * @returns {Array}
+ * @returns {Array<DropletBlock>}
  */
-function mergeFunctionsWithConfig(codeFunctions, dropletConfig, otherConfig, paletteOnly) {
+function filteredBlocksFromConfig(codeFunctions, dropletConfig, otherConfig, paletteOnly) {
```

Comment:
Once concern with the rename here is that `filteredBlocksFromConfig` suggests that it merely filters, but this function actually merges/extends/overrides config properties as well. I got burned by not understanding this last fall as I was doing the autocomplete work. So from readability POV, I'm not sure I like the new name. The old name was great either. Maybe `filterAndExtendBlocksFromConfig()` ?